### PR TITLE
Removed Doxygen characters from first line of documentation

### PR DESCRIPTION
**Pull request by gtribello**
In the nest and tutorials the first line of the documentation for actions appears in the tooltip that appears when you hover over the action name.  This line of text should thus provide a brief summary of what the action does.  To make this renderable this description should not include anything that needs to be rendered by Doxygen (i.e. equations and ref commands).  I found some places in PLUMED where people have included equations and ref commands in that first line of doxygen input.

This commit adds a check when the json syntax file is rendered to ensure folks do not have backslash characters in that first line of syntax. I think this should capture all the doxygen commands that people might use in these lines.

It also removes the few places where people have included equations and so on in that first line of the documentation

##### Description

See description above

##### Target release

I would like my code to appear in release v2.9

##### Type of contribution

- [x] changes to code or doc authored by PLUMED developers, or additions of code in the core or within the default modules
- [ ] changes to a module not authored by you
- [ ] new module contribution or edit of a module authored by you

##### Copyright

- [x] I agree to transfer the copyright of the code I have written to the PLUMED developers or to the author of the code I am modifying.

- [ ] the module I added or modified contains a `COPYRIGHT` file with the correct license information. Code should be released under an open source license. I also used the command `cd src && ./header.sh mymodulename` in order to make sure the headers of the module are correct. 

##### Tests

- [x] I added a new regtest or modified an existing regtest to validate my changes.
- [x] I verified that all regtests are passed successfully on [GitHub Actions](https://github.com/plumed/plumed2/actions).

### CODE REVIEW

**Review by GiovanniBussi — APPROVED** [review]
OK for me!

File: json/Makefile [hunk 1]
```diff
@@ -35,4 +35,4 @@ endif
 endif
 
 clean:
-	rm -fr syntax.json 
+	rm -fr syntax.json action_list
```
Comment (GiovanniBussi) [inline json/Makefile:38]: @gtribello is this change related to this PR?

Comment (gtribello) [reply json/Makefile:38]: Not necessarily.  I think it fixes a typo I made.  When you run that json makefile it creates two files: syntax.json and action list.  If you run make clean those two files should be deleted as these are what the makefile creates.

What should I do to resolve this type of problem in general.  Just commit and push it separately and say it fixes a typo?

Comment (GiovanniBussi) [reply json/Makefile:38]: It's fine I was just wondering.
